src/services/logging.service.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { Logger } from "./logging.service";
+import { mock, mockReset } from "jest-mock-extended";
+
+describe("LoggingService Unit Tests", () => {
+  let logger: Logger;
+  let internal: Console;
+
+  beforeAll(() => {
+    internal = mock<Console>();
+    logger = new Logger(internal);
+  });
+
+  afterEach(() => {
+    Logger.debug = true;
```

Comment:
This should be `false` (reset back to default). I guess this is the downside of using a truly Singleton pattern unless you get Jest to reset the module. Using the named export + default export approach mentioned in another comment would allow you to new up the `Logger` for each test without side-effects

Reply:
You are right. I was overcomplicating things. Switching right now.